src/Jackett.Common/Indexers/Shazbat.cs

```diff
@@ -49,141 +46,236 @@ private new ConfigurationDataBasicLoginWithRSS configData
             if (string.IsNullOrWhiteSpace(configData.RSSKey.Value))
                 throw new ExceptionWithConfigData("Failed to find RSS key.", configData);
+
             SaveConfig();
+
             return IndexerConfigurationStatus.RequiresTesting;
         }
 
         protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
         {
+            WebResult response;
+
             var releases = new List<ReleaseInfo>();
-            var queryString = query.GetQueryString();
-            WebResult results = null;
-            var searchUrls = new List<string>();
-            if (!string.IsNullOrWhiteSpace(query.SanitizedSearchTerm))
+            var searchUrls = new List<WebRequest>();
+
+            var hasGlobalFreeleech = false;
+
+            var searchTerm = query.SanitizedSearchTerm;
+            var term = FixSearchTerm(searchTerm);
+
+            var showTorrentsHeaders = new Dictionary<string, string>
+            {
+                { "Content-Type", "application/x-www-form-urlencoded" },
+                { "X-Requested-With", "XMLHttpRequest" },
+            };
+
+            var showTorrentsBody = new Dictionary<string, string>
+            {
+                { "portlet", "true" },
+                { "tab", "true" }
+            };
+
+            if (!string.IsNullOrWhiteSpace(term))
             {
-                var pairs = new Dictionary<string, string>
+                var searchBody = new Dictionary<string, string>
                 {
-                    {"search", query.SanitizedSearchTerm}
+                    { "search", term }
                 };
-                results = await RequestWithCookiesAndRetryAsync(
-                    SearchUrl, null, RequestType.POST, TorrentsUrl, pairs);
-                results = await ReloginIfNecessary(results);
+
+                response = await RequestWithCookiesAndRetryAsync(SearchUrl, method: RequestType.POST, referer: TorrentsUrl, data: searchBody);
+                response = await ReloginIfNecessaryAsync(response);
+
                 var parser = new HtmlParser();
-                var dom = parser.ParseDocument(results.ContentString);
+                var dom = parser.ParseDocument(response.ContentString);
+
+                hasGlobalFreeleech = dom.QuerySelector("span:contains(\"Freeleech until:\"):has(span.datetime)") != null;
+
+                releases.AddRange(ParseResults(response, query, searchTerm, hasGlobalFreeleech));
+
                 var shows = dom.QuerySelectorAll("div.show[data-id]");
-                foreach (var show in shows)
+                if (shows.Any())
                 {
-                    var showUrl = ShowUrl + show.GetAttribute("data-id");
-                    searchUrls.Add(showUrl);
+                    var showPagesFetchLimit = ShowPagesFetchLimit;
+
+                    if (showPagesFetchLimit <= 1 || showPagesFetchLimit > 5)
```

Comment:
```suggestion
                    if (showPagesFetchLimit < 1 || showPagesFetchLimit > 5)
```

Should this be `less than 1` rather than `less than or equal to 1` if 1 is an acceptable value?